pkg/alertmanager/multitenant.go

```diff
@@ -60,6 +61,8 @@ var (
 	errInvalidExternalURLMissingHostname   = errors.New("the configured external URL is invalid because it's missing the hostname")
 	errZoneAwarenessEnabledWithoutZoneInfo = errors.New("the configured alertmanager has zone awareness enabled but zone is not set")
 	errNotUploadingFallback                = errors.New("not uploading fallback configuration")
+	errDefaultTemplateInvalid              = errors.New("-alertmanager.default-template validation failed")
```

Comment:
nit: feels weird to put the option name in the error because we don't do that elsewhere.